test/framework/module_generator.py

```diff
@@ -50,10 +50,10 @@
 
 
 class ModuleGeneratorTest(EnhancedTestCase):
-    """ testcase for ModuleGenerator """
+    """ testcase for ModuleGeneratorTcl """
```

Comment:
I need to think about how to do this...

I *think* we want to run the whole test suite with Tcl syntax, and Lua syntax, similar to what we do with Lmod and Tcl env mods.

So, some of these tests would have to have a Tcl and Lua variant, if they hardcode any module file syntax things.